transport/http/handler.go

```diff
@@ -244,17 +259,14 @@ func (h handler) createSpan(ctx context.Context, req *http.Request, treq *transp
 }
 
 var (
-	_ transport.ExtendedResponseWriter     = (*responseWriter)(nil)
```

Comment:
Why these changes a rolled back to a dev-branch version? (From here and until the rest of file)